pontoon/base/tests/test_models.py

```diff
@@ -80,6 +84,102 @@ def test_repository_for_path(self):
         path = os.path.join(repo2.checkout_path, 'foo', 'bar')
         assert_equal(project.repository_for_path(path), repo2)
 
+    def locales_parts_stats_helper(self):
+        locale = LocaleFactory.create()
+        locale_other = LocaleFactory.create()
+        project = ProjectFactory.create(locales=[locale, locale_other])
+        resource = ResourceFactory.create(
+            project=project,
+            path='/main/path.po'
+        )
+        EntityFactory.create(resource=resource)
+        StatsFactory.create(resource=resource, locale=locale)
+
+        return locale, locale_other, project, resource
+
+    def test_locales_parts_stats_no_page_one_resource(self):
+        """
+        Return empty list.
```

Comment:
It's useful to also include what conditions should cause `locales_parts_stats` to return an empty list.